website/content/plugins/drivers/exec2.mdx

```diff
@@ -305,6 +305,47 @@ driver.exec2.unveil.defaults    = true
 driver.exec2.unveil.tasks       = true
 ```
 
+### Benefits of exec2 Over exec
+
+#### Security Benefits
+
+##### Enhanced Filesystem Isolation with Landlock LSM
+- **Landlock LSM**: The `exec2` driver leverages the Landlock Linux Security Module (LSM) for filesystem isolation, which is more secure than the traditional `chroot` mechanism used by the `exec` driver.
```

Comment:
I think there is a misunderstanding about the value prop of exec2 - it's primarily around the performance of an exec-like driver, not the absolute security benefits. exec and exec2 provide approximately the same security - if anything exec2 is more lenient in that it allows the cluster operator to open specific parts of the host filesystem to the tasks whereas exec is fully isolated apart from mounted volumes. where exec2 shines is that it will never create a chroot in which to run each task, which can be a very slow operation on many systems depending on filesystem performance (think network drives) and the amount of content in the chroot being created.

Reply:
Hey @shoenig, for the security benefits section I was mostly attempting to unpack the first half of the "Chroot Challenges" section of [NMD-190](https://docs.google.com/document/d/1cDH-qy0nb6Fc4PyEEqs_HfoNv_BXUwes7eAr2MqE4os/edit?usp=sharing).  Mainly contrasting these two claims:

- "The fundamental problem with chroots is this: they are not secure."
- "A process that has been locked (by Landlock LSM) cannot be unlocked, even if the process is being run as root"

Reply:
@shoenig Could you propose a different approach here or way to phrase the benefits of exec2 instead?